src/pages/en/reference/api-reference.md

```diff
@@ -91,13 +91,13 @@ const data = await Astro.glob<CustomDataFile>('../data/**/*.js');
 
 ### `Astro.request`
 
-`Astro.request` is a standard [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) object. It can be used to get the `url`, `headers`, `method`, and even body of the request. Use `new URL(Astro.request.url)` to get a URL object.
+`Astro.request` is a standard [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) object. It can be used to get the `url`, `headers`, `method`, and even body of the request. 
```

Comment:
Do we want to call out that this of most use with SSR? Push folks towards `Astro.url` for most common uses?

```suggestion
`Astro.request` is a standard [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) object. It can be used to get the `url`, `headers`, `method`, and even body of the request. This is most useful for sites using [SSR](/en/guides/server-side-rendering/) that want to handle custom requests.
```

Reply:
I don't think this is true! It should be equally useful for both SSR and SSG.